editor/modes/visual-editor/block.js

```diff
@@ -51,13 +53,28 @@ import {
 	getMultiSelectedBlockUids,
 } from '../../selectors';
 
-const { BACKSPACE, ESCAPE, DELETE, ENTER } = keycodes;
+const { BACKSPACE, ESCAPE, DELETE, ENTER, TAB, F10, UP, LEFT, DOWN, RIGHT } = keycodes;
 
 function FirstChild( { children } ) {
 	const childrenArray = Children.toArray( children );
 	return childrenArray[ 0 ] || null;
 }
 
+function queryFirstTabbableChild( elem ) {
+	let i = 0;
+	for ( i = 0; i < elem.childNodes.length; i++ ) {
+		const tabbableNode = queryFirstTabbable( { context: elem.childNodes[ i ] } );
```

Comment:
That includes the element itself in the query so I can't use it like that. I tried passing an array but then I found that it silently discards all but the first element of the array (same with a selector despite using querySelectorAll internally which was quite confusing).